components/Navigation/index.js

```diff
@@ -1,16 +1,14 @@
 import { theme } from "@/styles";
 import styled from "styled-components";
 
-export default function Navigation() {
+export default function Navigation({ onCreateClick }) {
   return (
     <StyledNavigation>
       <StyledNavButtonBox>
         <StyledSideButton onClick={() => (window.location.href = "/joined")}>
           XX
         </StyledSideButton>
-        <StyledCenterButton onClick={() => (window.location.href = "/create")}>
-          create
-        </StyledCenterButton>
+        <StyledCenterButton onClick={onCreateClick}>create</StyledCenterButton>
         <StyledSideButton onClick={() => (window.location.href = "/calendar")}>
```

Comment:
this should not be a button, but a Link, since you navigate to another page.

It would be a button if some logic would have to be applied before navigating.

Reply:
Before we wrapped the Button inside a Link but we got the feedback that it's semantically incorrect. 
We use a Button for styling reasons. But we should use and style the Link instead?

Reply:
We will fix that in another userstory.

Reply:
no, fix this now. This is an easy fix